assets/pages/oneoff-contributions/oneOffContributionsReducer.js

```diff
@@ -53,6 +53,9 @@ function createOneOffContribReducer(amount: number) {
       case 'CHECKOUT_ERROR':
         return Object.assign({}, state, { error: action.message });
 
+      case 'CHECKOUT_SUCCESS':
+        return Object.assign({}, state, { paymentComplete: true });
```

Comment:
Does `paymentComplete` need to be added to the `State` type in this file? I think flow would throw an error here if we were using an [exact object type](https://flow.org/en/docs/types/objects/#toc-exact-object-types) for `State` (i.e. `||`)?